src/components/Chat/InputMessageV2.tsx

```diff
@@ -194,24 +404,47 @@ const InputMessageV2 = ({
             labelLeft={!inputFocus ? 'Anonymity' : undefined}
             styleLabelLeft={[
               styles.labelToggle,
-              {color: type === 'ANONYMOUS' ? colors.gray : colors.blue}
+              {color: type === ANONYMOUS ? colors.gray : colors.blue}
             ]}
-            onValueChange={toggleChnage}
-            value={type === 'ANONYMOUS'}
+            onValueChange={toggleChange}
+            value={type === ANONYMOUS}
+            onValueChange={toggleChange}
+            value={type === ANONYMOUS}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onValueChange` prop and `value` prop of the ToggleSwitch component are duplicated. One of each should be removed.



<!-- This is an auto-generated comment by OSS CodeRabbit -->